<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Carac</class>
 <widget class="QWidget" name="Carac">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>93</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgba(211, 215, 207,180)</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="caracBox">
       <property name="toolTip">
        <string extracomment="youpi je suis une tooltip"/>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="AspectRatioLabel" name="imageCarac">
            <property name="toolTip">
             <string>youpi je suis une tooltip !!</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="labelValeur">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="jaugeCarac">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="value">
               <number>-1</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
              <property name="format">
               <string>%p</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AspectRatioLabel</class>
   <extends>QLabel</extends>
   <header>aspectratiolabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
